src/components/ui/app-sidebar.tsx

```diff
@@ -87,6 +88,11 @@ const data = {
         },
       ],
     },
+    {
+      title: 'Recomendaciones',
+      url: '/recomendaciones',
+      icon: Star,
+    },
```

Comment:
Le agregaría `Software` en el título asi queda claro que son recomendaciones de eso. No te olvides de actualizar la `url` cuando le cambies el nombre a la ruta por `software-recomendado`
```suggestion
    {
      title: 'Software recomendado',
      url: '/software-recomendado',
      icon: Star,
    },
```

Reply:
Gracias por las recomendaciones agustin